### PR TITLE
cherry pick and merge address state

### PR DESCRIPTION
**Pull request by ameyachavan007**
cherry picked from [f75edaa](https://github.com/precision-sustainable-ag/dst-selector/pull/377/commits/f75edaa92159266920b8152f2ad0b781f6d64df7)

completed address state migration
errors to discuss -  GET https://api.covercrop-selector.org/v1/states/undefined/dictionary?regions=undefined 422

### CODE REVIEW

**Comment by mikahpinegar** [comment]
forecast not updating with location change

**Comment by mikahpinegar** [comment]
weather not updating with location change

**Comment by mikahpinegar** [comment]
going from location tab to soils tab and then back to location causes infinite loop of api calls

File: src/reduxStore/addressSlice.js [hunk 1]
```diff
@@ -82,13 +77,22 @@ export const updateLastZone = (value) => ({
...
+// export const updateMarker = (value) => {
+//     return {
+//         type: 'UPDATE_MARKER',
+//         payload: {
+//             value,
+//         },
+//     };
+// };
+
+const addressReducer = (state = initialState, action) => {
```
Comment (Adamws33) [inline src/reduxStore/addressSlice.js:89]: the default parameters should be last (action, state = initialState)

**Review by mikahpinegar — APPROVED** [review]

File: src/reduxStore/addressSlice.js (hunk 1 above)
Comment (mikahpinegar) [reply src/reduxStore/addressSlice.js]: good catch, i just pushed a fix

**Review by Adamws33 — APPROVED** [review]

**Pull request merged**